{{cookiecutter.project_name}}/docker/django/Dockerfile

```diff
@@ -8,6 +8,7 @@ LABEL vendor="wemake.services"
 
 ARG DJANGO_ENV
 
+ENV USER=web USER_ID=1000 USER_GID=1000
```

Comment:
Let's add some docs about why do we use `1000` and `web`

Reply:
Web is the username which you define it. I just move to ENV. 
By default, Linux systems automatically assign UIDs and GIDs to new user accounts in numerical order starting at 1000. So in the Linux system your user id is 1000. I decided to use it by default.

Reply:
Please, add this to the comment docs

Reply:
Sure. But I'm not sure where to add it.